lib/ci/container/ephemeral.rb

```diff
@@ -17,10 +18,15 @@ class << self
 
     def stop(options = {})
       super(options)
-      kill!(options)
+      kill!(options) if running?
       rescued_remove
     end
 
+    def running?
+      state = json.fetch('State')
+      raise EphemeralContainerUnhandledState unless %w(exited running).include? state.fetch('Status')
```

Comment:
rubocop

also, this is code and not a dsl so use brackets

also, comment on why this is necessary

also, so, I am not allowed to have containers on my local system if I want to be able to run the pangea tests?